db/models/feedback.js

```diff
@@ -0,0 +1,18 @@
+'use strict'
+
+const Sequelize = require('sequelize')
+const db = require('APP/db')
+
+const Feedback = db.define('feedback', {
+  date: {
+    type: Sequelize.DATE,
+    defaultValue: Sequelize.NOW
+  },
+  // e-mail not validated here because email is not required
+  // email should be checked for validity on front-end
```

Comment:
Oops, yeah I was thinking that that would require the email field not be null, also.